src/Accounts/Accounts/Models/RMProfileClient.cs

```diff
@@ -412,13 +415,12 @@ public bool TryGetSubscriptionById(string tenantId, string subscriptionId, out I
             return subscription != null;
         }
 
-        public bool TryGetSubscriptionByName(string tenantId, string subscriptionName, out IAzureSubscription subscription)
+        public bool TryGetSubscriptionByName(string tenantId, string subscriptionName, out List<IAzureSubscription> subscriptions)
```

Comment:
use interface whenever possible
```suggestion
        public bool TryGetSubscriptionByName(string tenantId, string subscriptionName, out IEnumerable<IAzureSubscription> subscriptions)
```